balsamiq.md

```diff
@@ -0,0 +1,131 @@
+# Case Study - Balsamiq Docs: Static Doesn't Have to Be a Limitation
+
+Leon Barnard describes how the second version of the Balsamiq Docs static site went beyond purely functional to do things he didn't think markdown and static sites could do.
+
+
+## Background
+
+A year and a half ago [we ditched our flaky content management system](https://blog.balsamiq.com/new-documentation-site/) and converted our documentation site over to a "docs like code" system using [Hugo](http://gohugo.io/), [Gulp](http://gulpjs.com/), and [GitHub](https://github.com/), with content written in markdown. It was a long process and we were happy just to see it up and running.
+
+After the dust settled, we started imagining what we wanted to do for the next version and realized that our system had limitations that we would have to overcome. This is a story of three challenges and what we did to solve them.
+
+***Note:*** *These are specific to [Hugo](http://gohugo.io/), the static site generator we use, but should be adaptable to others.*
+
+## Challenge #1: Documenting multiple product versions
+
+Our product, [Balsamiq Mockups](https://balsamiq.com/), comes in multiple "flavors," as we often call them. There are two versions of the core product (the wireframe editor) that are sold as: a stand-alone desktop version, a hosted web version, and plugins for Google Drive and Atlassian Confluence and JIRA. 
+
+![](https://media.balsamiq.com/images/docslikecode/select-product.png)
```

Comment:
Nice, I'll probably download and upload to the docslikecode site if that's okay with you.